static/app/views/eventsV2/table/arithmeticInput.tsx

```diff
@@ -307,36 +307,39 @@ type TermDropdownProps = {
 function TermDropdown({isOpen, optionGroups, handleSelect}: TermDropdownProps) {
   return (
     <DropdownContainer isOpen={isOpen}>
-      <DropdownItemsList>
-        {optionGroups.map(group => {
-          const {title, options} = group;
-          return (
-            <Fragment key={title}>
-              <ListItem>
-                <DropdownTitle>{title}</DropdownTitle>
-              </ListItem>
-              {options.map(option => {
-                return (
-                  <DropdownListItem
-                    key={option.value}
-                    className={option.active ? 'active' : undefined}
-                    onClick={() => handleSelect(option)}
-                    // prevent the blur event on the input from firing
-                    onMouseDown={event => event.preventDefault()}
-                    // scroll into view if it is the active element
-                    ref={element =>
-                      option.active && element?.scrollIntoView?.({block: 'nearest'})
-                    }
-                  >
-                    <DropdownItemTitleWrapper>{option.value}</DropdownItemTitleWrapper>
-                  </DropdownListItem>
-                );
-              })}
-              {options.length === 0 && <Info>{t('No items found')}</Info>}
-            </Fragment>
-          );
-        })}
-      </DropdownItemsList>
+      {isOpen && (
```

Comment:
Why does it need this?

Reply:
because otherwise everything gets rendered in the DOM  when it shouldn't

Reply:
Is it not hidden in some way?

Reply:
yes through CSS

Reply:
Like how was this working before? haha

Reply:
Ahhh hmm. Should we remove the CSS then?

Reply:
https://github.com/getsentry/sentry/pull/41007#discussion_r1013905715

Reply:
we could, but there is a border and I was afraid that I could cause some jumping effect... maybe I can better take a look into it in a follow up